pylint/config/utils.py

```diff
@@ -203,9 +203,8 @@ def _enable_all_extensions(run: Run, value: str | None) -> None:
 PREPROCESSABLE_OPTIONS: dict[
     str, tuple[bool, Callable[[Run, str | None], None], int]
 ] = {  # pylint: disable=consider-using-namedtuple-or-dataclass
-    # pylint: disable=wrong-spelling-in-comment
     # Argparse by default allows abbreviations. It behaves differently
-    # if you turn this off, so we also turn it on. We mimick this
+    # if you turn this off, so we also turn it on. We mimic this
```

Comment:
Sorry about the unrelated change but I don't know why suddenly pre-commit pylint was not happy about this.

Reply:
I must have a dictionary with both mimic and mimick.

Reply:
We might want to add to our own dictionary instead of disabling because the ``useless-suppression`` it will get annoying fast.

Reply:
This is because you removed a `disable` on L206.

Reply:
No, I had the useless suppression, thougth it was a spellcheck issue, "fixed" "mimick" (suggestion from IDE) then still had an useless-suppression, and removed the disable. I think it's the dict that has "mimick" in it on ubuntu.

Reply:
Let's add `useless-suppression` to the disable there then. Because the other stuff in that comment still needs to be disabled.